src/test/java/com/scottlogic/filter/FilterAuthorTest.java

```diff
@@ -0,0 +1,30 @@
+package com.scottlogic.filter;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.utils.UserPostFilterData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class FilterAuthorTest {
+
+    FilterAuthor filterAuthorTest = new FilterAuthor();
```

Comment:
You can just call this filterAuthor, it's not a test but the actual thing - this file is your test and it is using the actual thing.